include/raykernels/PICKernelBase.h

```diff
@@ -0,0 +1,42 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
```

Comment:
```suggestion
//* This file is part of FENIX: Fusion Energy Integrated Multiphys-X,
//* A multiphysics application for modeling plasma facing components
//* https://github.com/idaholab/fenix
//*
//* FENIX is powered by the MOOSE Framework
//* https://www.mooseframework.inl.gov
//*
//* Licensed under LGPL 2.1, please see LICENSE for details
//* https://www.gnu.org/licenses/lgpl-2.1.html
```

We should create a FENIX file header instead of using the MOOSE one, and then replace all instances of this header in the PR. I've made an attempt at one here. Any thoughts @simopier @loganharbour?

Reply:
Fine by me. remove the redundant www.mooseframework.inl.gov -> mooseframework.inl.gov

Reply:
Sorry, I don't seem to get github notifications. 

Yes, it looks good to me.